news/2021-11-04-skinners-bulletin-august-2021.md

```diff
@@ -0,0 +1,84 @@
+
+The use of arrows and rhomboidal shapes coloured blue and yellow beautifully complement each other and the dotted texture at the background, sending out minimalist yet graffiti-esque vibes. Immediately drawing attention right at the song selection menu, the gfx work covering the osu! cookie parting from the rest of the screen with its unique presentation of the character the skin is revolving around. On the top section of the screen, the use of arrows points out the grouping/sorting menu on the right side very nicely, where the `No Grouping` option stands out with its contrastive red colour without seeming misplaced at all.
+
+The dotted texture of menu sections and buttons covers gameplay elements as well, even hitcircles, which shows us how much delicacy went into details. The arrows and dotted pattern follow us to the pause and fail menus - in other words, *water-break sections* - which are just another example of Inflique's craftsmanship. Ultimately, Ai Hayasaka v3.0 is one-of-a-kind anime-themed skin that manages to shine amongst its counterparts, thanks to the flawless and consistent design carried out by Inflique. No more words to say; just have a look at this showcase video below and give it a try. Oh, and don't forget to stay hydrated!
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/2COogi0XYqY" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Tomioka Giyū
+
+Splashing our bulletin like a tsunami, [Tomioka Giyū](https://osu.ppy.sh/community/forums/topics/1405313), by the very hands of [Kim Minnie](https://osu.ppy.sh/users/3858685), goes further than just being an anime-themed skin by featuring an oil-painting-like look. Brushed into existence with delicacy, Giyū goes down the less-traveled path embracing a flashy and artsy theme, which is quite the opposite of the current minimalism trend.
+
+Before we get into the details on the song selection menu, we highly advise you to use your life vest in case of an emergency, as these deadly waves look like a health hazard! Putting aside the wavy design though, the inkblots appearing in many places, mainly on the buttons and whatnot, reinforce the artsiness to another level. On the background of the game mode selection screen, the fabulous graphic of Tomioka himself greets us with a pose wielding his katana fading into the azureness of the sea. As for the game modifiers, they choose to entertain the portraits of the characters from the anime, where the respective mod is mentioned on the bottom-right corner.
```

Comment:
```suggestion
Splashing our bulletin like a tsunami, [Tomioka Giyū](https://osu.ppy.sh/community/forums/topics/1405313), by the very hands of [Kim Minnie](https://osu.ppy.sh/users/3858685), goes further than just being an anime-themed skin by featuring an oil-painting-like look. Brushed into existence with delicacy, Giyū goes down the less-traveled path embracing a flashy and artsy theme, which is quite the opposite of the current minimalism trend. But before we get into the details on the song selection menu, we highly advise you to use your life vest in case of an emergency, as these deadly waves look like a health hazard!

Putting the wavy design aside, the inkblots appearing in many places, mainly on the buttons and whatnot, reinforce the artsiness to another level. On the background of the game mode selection screen, a fabulous graphic of Tomioka himself greets us with a pose wielding his katana, fading into the azure sea. As for the game modifiers, they choose to entertain portraits of the characters from the anime, with the respective mods mentioned in the bottom-right corners.
```